src/utils/NumberUtils.h

```diff
@@ -0,0 +1,205 @@
+// SPDX-License-Identifier: BSD-3-Clause
+// Copyright Contributors to the OpenColorIO Project.
+
+#ifndef INCLUDED_NUMBERUTILS_H
+#define INCLUDED_NUMBERUTILS_H
+
+#if defined(_MSC_VER)
+#define really_inline __forceinline
+#else
+#define really_inline inline __attribute__((always_inline))
+#endif
+
+#ifdef USE_FAST_FLOAT
+#include <fast_float/fast_float.h>
+#endif
+
+#include <cstdlib>
+#include <locale>
+#include <system_error>
+#include <type_traits>
+
+namespace NumberUtils
```

Comment:
The header code must be in the `OCIO_NAMESPACE` namespace.

Reply:
Is this correct? 

https://github.com/AcademySoftwareFoundation/OpenColorIO/blob/64dccadba11826a2358bf192519563568462cf08/src/utils/StringUtils.h#L18

Reply:
No. I found no historical reasons to not have the OCIO namespace so, I assume that we missed it.